### PR TITLE
Add optional fields child-client model

### PR DESCRIPTION
**Pull request by RamuKammagani-osi**

### CODE REVIEW

File: lib/child_clients/child_client.rb [hunk 1]
```diff
@@ -92,11 +94,36 @@ class ChildClient < Dry::Struct
...
+    attribute :confidentiality_action_date, Types::String.optional
+    attribute :death_date, Types::String.optional
+    attribute :death_reason, Types::String.optional
+    attribute :health_txt, Types::String.optional
+    attribute :mother_parental_term_date, Types::String.optional
+    attribute :father_parental_term_date, Types::String.optional
+    attribute :death_place, Types::String.optional
+    attribute :email_addr, Types::String.optional
+    attribute :adjudicated_delinquent_indicator, Types::String.optional
+    attribute :eth_unable_to_determin_code, Types::String.optional
```
Comment (kelfink) [inline lib/child_clients/child_client.rb]: eth_unable_to_determine_code. let's add the 'e' unless this is matching a (strange?) API response

File: lib/child_clients/child_client.rb [hunk 2]
```diff
@@ -92,11 +94,36 @@ class ChildClient < Dry::Struct
...
+    attribute :hisp_unable_to_determinate_reason_code, Types::String.optional
```
Comment (kelfink) [inline lib/child_clients/child_client.rb]: hisp_unable_to_determine_reason_code.  is a better phrase.  Unless this is an API requirement

File: spec/lib/child_clients/child_client_spec.rb [hunk 3]
```diff
@@ -150,6 +150,44 @@ module ChildClients
...
+      it { is_expected.to have_attribute(:death_reason, Types::String.optional) }
+      it { is_expected.to have_attribute(:health_txt, Types::String.optional) }
+      it { is_expected.to have_attribute(:mother_parental_term_date, Types::String.optional) }
+      it { is_expected.to have_attribute(:father_parental_term_date, Types::String.optional) }
+      it { is_expected.to have_attribute(:death_place, Types::String.optional) }
+      it { is_expected.to have_attribute(:email_addr, Types::String.optional) }
+      it {
+        is_expected.to have_attribute(:adjudicated_delinquent_indicator, Types::String.optional)
+      }
+      it { is_expected.to have_attribute(:eth_unable_to_determin_code, Types::String.optional) }
```
Comment (kelfink) [inline spec/lib/child_clients/child_client_spec.rb]: determine. with an 'e' please

File: spec/lib/child_clients/child_client_spec.rb [hunk 4]
```diff
@@ -150,6 +150,44 @@ module ChildClients
...
+
+      it {
+        is_expected
+          .to have_attribute(:hisp_unable_to_determinate_reason_code, Types::String.optional)
```
Comment (kelfink) [inline spec/lib/child_clients/child_client_spec.rb]: determinate should be simply determine

File: lib/child_clients/child_client.rb (hunk 1 above)
Comment (RamuKammagani-osi) [reply lib/child_clients/child_client.rb]: That's how i'm getting the response from API.

File: lib/child_clients/child_client.rb (hunk 2 above)
Comment (RamuKammagani-osi) [reply lib/child_clients/child_client.rb]: Same here, Whatever i add here it should match with the response, I can't change it unless it is changed in API

File: spec/lib/child_clients/child_client_spec.rb (hunk 3 above)
Comment (RamuKammagani-osi) [reply spec/lib/child_clients/child_client_spec.rb]: same here

File: spec/lib/child_clients/child_client_spec.rb (hunk 4 above)
Comment (RamuKammagani-osi) [reply spec/lib/child_clients/child_client_spec.rb]: Same here

**Pull request merged**